build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.0'
+        classpath 'com.android.tools.build:gradle:2.4.0-alpha7'
```

Comment:
I have to using this to demonstrate compatibility (or lack there of) between the latest versions of kapt, AGP, databinding, etc. But I understand if you don't want non-stable versions on master. 

#22 includes this as well, so I will close both PRs and open one with stable dependency versions. See #23